### ISSUE TITLE
README snippet typo

### ISSUE THREAD

**Issue by nahue-f** [delvedor/find-my-way#87]
Hi!

There is an error in this snippet:  
```javascript
router.on('GET', '/example', { version: '1.2.0' }, (req, res, params) => {
  res.send('Hello from 1.2.0!')
})

router.on('GET', '/example', { version: '2.4.0' }, (req, res, params) => {
  res.send('Hello from 2.4.0!')
})

// The 'Accept-Version' header could be '1.2.0' as well as '*', '2.x' or '2.4.x'
```

`res.send` it's not a valid function, since `send` is an Express and/or Restify API. It should be `res.end`.

**Comment by delvedor** [comment]
Nice catch! Would you like to send a pr to fix it?

**Comment by nahue-f** [comment]
Sure!

**Comment by mcollina** [comment]
Fixed in #88
